decentraland/unity-renderer
# feat: block unfriend confirm

## What does this PR change?

Displays a new confirmation dialog before unfriending or blocking another user.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/block-unfriend-confirm
2. Open the friend list
3. Click the three dots, open the contextual menu and then hit unfriend. A dialog should be opened
4. Open the contextual menu and hit block. A dialog should be opened
5. Open the passport of a friend, click unfriend or block. A dialog should be opened
6. Open a private chat window with a friend
7. Click three dots at the header, hit unfriend or block. A dialog should be opened

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
